### DISCUSSION TITLE
Is it possible to Use Xpaths for webview testing using Patrol $.native ?

### DISCUSSION THREAD

**Discussion by RashmiShashindra** [leancodepl/patrol#1842] [Q&A]
I am curious about if we can use Xpaths for the webview testing in flutter integration tests using patrol? Issue I faced was, some of the WebViews we don't have control over that hence cannot capture from className, resourceID etc.

```
static final Selector element = Selector(className: 'element_classname');
$.native.tap(element)
```
I have checked and Selector has only below methods.

```
class Selector with EquatableMixin {
  Selector({
    this.text,
    this.textStartsWith,
    this.textContains,
    this.className,
    this.contentDescription,
    this.contentDescriptionStartsWith,
    this.contentDescriptionContains,
    this.resourceId,
    this.instance,
    this.enabled,
    this.focused,
    this.pkg,
  });
```

Just curious if we have any other way to use Xpaths in WebViews or it is not possible at all? :)

**Comment by bartekpacia** [comment]
No, we don't support xpaths.
